app/src/main/res/values/strings.xml

```diff
@@ -654,6 +654,9 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_bikeParkingCapacity_title">"How many bikes can be parked here?"</string>
     <string name="quest_bikeParkingCapacity_hint">"Note that most normal stands can be used from both sides to park each one bike."</string>
 
+    <string name="quest_air_pump_bicycle_title">Is a working bicycle air pump available to everybody having access here?</string>
```

Comment:
Maybe "Can anyone use an air pump provided here?"

Or something similar?

(not a native speaker speaker)

Reply:
it turns out that `access=customers` is invalid usage here (e747679d7331e5b3e463fe955d2f2091deadca0d), so currently suggested string is:
`Is a working bicycle air pump available here for everyone?`

That looks passable to me, but I'd be happy for improvements, provided that it keeps clear the following information:

- it is about `bicycle air pump` (and not some other pump, like oil pump for hydraulic bicycle brakes, or any other pump not usable for bicycle tires) 
- it is in working condition (important, as in some countries like my own it cannot be assumed at all :cry: )
- it is available to any member of general public (and not only to customers or depending on goodwill of the owner etc.)

Reply:
(native speaker here)
How about "Is there a working bicycle air pump available for everyone here?" (although bicycle air pump sounds a bit odd as I would just call it a bicycle pump)
and "Is there an air compressor available here?" reads better

Reply:
thanks @arrival-spring , I've changed them as you suggested then: bf5472f694346494cb81b7ee72a2f35731bd2d33! 

I did actually try similar string for air compressor before, but in the end, while it sounded OK to me, it looked highly suspicious to have both `there` _and_ `here` referring to same location in same sentence, so I decided to remove one of them. So it's great to have your confirmation that it is actually OK.

I take solace in the fact that any English-speaking person would have even harder time learning proper Croatian :laughing: